flash_xc3.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 set -e
-set -x
 
-/sbin/fxload -t fx2 -I /opt/Xilinx/14.7/ISE_DS/ISE/bin/lin64/xusb_xp2.hex -D /dev/bus/usb/001/*`cat /sys/bus/usb/devices/1-7/devnum`
-sleep 7
-../xc3sprog/build/xc3sprog -c xpc -m /opt/Xilinx/14.7/ISE_DS/ISE/xbr/data -v build/mirny.jed:w
+XC3SPROG=xc3sprog
+CABLE=$([[ -z "$1" ]] && echo "xpc" || echo "$1")
```

Comment:
There is some simpler and standard way to default a variable in (ba)sh.

Reply:
Indeed, thanks.